partner_industry_secondary/models/res_partner_industry.py

```diff
@@ -28,11 +29,25 @@ def get_names(cat):
             """Return the list [cat.name, cat.parent_id.name, ...]"""
             res = []
             while cat:
-                res.insert(0, cat.name)
+                res.append(cat.name)
                 cat = cat.parent_id
             return res
 
-        return [(cat.id, " / ".join(get_names(cat))) for cat in self]
+        if (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("partner_industry_secondary.display_last_child_first")
```

Comment:
thought: don't you believe the "children first" format would be a better default value?

cc @rafaelbn